doc/src/Transactions.dox

```diff
@@ -35,7 +35,7 @@
  *    manager.startTransaction(context)
  *    try:
  *        # Create the shot first, so we can then publish the clips to it
- *        shot_ref = manager.register([shot.name], [shot.metadata],
+ *        shot_ref = manager.register([shot.name], [shot.attributes],
  *                                    [shot.spec], [entity_ref], context)[0]
```

Comment:
[consider]/[plan] Not caused by this PR. This doesn't match the signature of `register`, `attributes` should be a keyword argument.

Reply:
#129